tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -4110,6 +4116,12 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_MultispeedPerformance)
               thisSys->m_CoolVolumeFlowRate[10] * state->dataUnitarySystems->designSpecMSHP[0].coolingVolFlowRatio[6]);
     EXPECT_EQ(thisSys->m_HeatVolumeFlowRate[7],
               thisSys->m_HeatVolumeFlowRate[10] * state->dataUnitarySystems->designSpecMSHP[0].heatingVolFlowRatio[6]);
+
+    // test system no load air volume flow rate
+    Real64 result2_expected_NoLoadAirVolFlow =
+        state->dataUnitarySystems->designSpecMSHP[0].noLoadAirFlowRateRatio * std::min(thisSys->m_MaxCoolAirVolFlow, thisSys->m_MaxHeatAirVolFlow);
+    EXPECT_EQ(0.05, state->dataUnitarySystems->designSpecMSHP[0].noLoadAirFlowRateRatio);
+    EXPECT_EQ(result2_expected_NoLoadAirVolFlow, thisSys->m_MaxNoCoolHeatAirVolFlow);
 }
```

Comment:
Modified the same existing unit test to demonstrate the fix for a different flow rate.